### PR TITLE
Gatt/gap fixes

### PR DESCRIPTION
**Pull request by szymon-czapracki**
Various GAP/GATT test case fixes and refactor.

### CODE REVIEW

**Comment by kolodgrz** [comment]
I'll suggest to squash and make only 2 commits: use generic wid handler for GATT CL/(GAW/GAS) test cases, use generic wid handler for GAP/DISC/(NONM/LIMM). If those commits are related to entire group of test cases.

File: ptsprojects/zephyr/gatt_wid.py [hunk 1]
```diff
@@ -424,6 +424,37 @@ def hdl_wid_69(desc):
...
+    pattern = re.compile("'([0-9a-fA-F]+)'")
+    params = pattern.findall(desc)
+    if not params:
+        logging.error("parsing error")
+        return False
+
+    handle = params[0]
+    size = int(params[1])
+
+    btp.gattc_write_without_rsp(btp.pts_addr_type_get(None), btp.pts_addr_get(None), handle, '12', size)
```
Comment (tedd-an) [inline ptsprojects/zephyr/gatt_wid.py]: Please keep the line 80 column max.

File: ptsprojects/zephyr/gatt_wid.py [hunk 2]
```diff
@@ -424,6 +424,37 @@ def hdl_wid_69(desc):
...
+    pattern = re.compile("'([0-9a-fA-F]+)'")
+    params = pattern.findall(desc)
+    if not params:
+        logging.error("parsing error")
+        return False
+
+    handle = params[0]
+    size = int(params[1])
+
+    btp.gattc_write(btp.pts_addr_type_get(None), btp.pts_addr_get(None), handle, '12', size)
```
Comment (tedd-an) [inline ptsprojects/zephyr/gatt_wid.py]: Please keep the line 80 column max.

File: ptsprojects/zephyr/gatt_wid.py [hunk 3]
```diff
@@ -448,6 +479,29 @@ def hdl_wid_75(desc):
...
+    pattern = re.compile("'([0-9a-fA-F]+)'")
+    params = pattern.findall(desc)
+    if not params:
+        logging.error("parsing error")
+        return False
+
+    handle = params[0]
+    size = int(params[1])
+
+    btp.gattc_write_long(btp.pts_addr_type_get(None), btp.pts_addr_get(None), handle, 0, '12', size)
```
Comment (tedd-an) [inline ptsprojects/zephyr/gatt_wid.py]: Please keep the line 80 column max.

File: ptsprojects/zephyr/gatt_wid.py [hunk 4]
```diff
@@ -448,6 +479,29 @@ def hdl_wid_75(desc):
...
+    btp.gattc_write_long_rsp()
+
+    return True
+
+
+def hdl_wid_82(desc):
+    btp.gattc_write(btp.pts_addr_type_get(None), btp.pts_addr_get(None), '0100', '12', 1)
```
Comment (tedd-an) [inline ptsprojects/zephyr/gatt_wid.py]: Please keep the line 80 column max.

File: ptsprojects/zephyr/gatt_wid.py (hunk 3 above)
Comment (szymon-czapracki) [reply ptsprojects/zephyr/gatt_wid.py]: Done

File: ptsprojects/zephyr/gatt_wid.py (hunk 2 above)
Comment (szymon-czapracki) [reply ptsprojects/zephyr/gatt_wid.py]: Done

File: ptsprojects/zephyr/gatt_wid.py (hunk 1 above)
Comment (szymon-czapracki) [reply ptsprojects/zephyr/gatt_wid.py]: Done

**Pull request merged**